net/atlas-sw-probe/Makefile

```diff
@@ -7,15 +7,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=atlas-sw-probe
-PKG_VERSION:=5040
+PKG_VERSION:=5080
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/RIPE-NCC/ripe-atlas-software-probe.git
 PKG_SOURCE:=ripe-atlas-software-probe-$(PKG_VERSION).tar.gz
```

Comment:
Not needed.

Reply:
Yeah, but I haven't touched it. :-( We should do it in separate PR as your PR was merged, and usually, it is a good idea to have the same version of busybox for atlas-sw-probe to remain the compatibility and as it seems to be tested with that version.

Ref: https://github.com/RIPE-NCC/ripe-atlas-software-probe, and there is used 2.6.2 version, not 2.6.3, so hopefully, it will not cause any issues.